README.md

```diff
@@ -46,6 +46,28 @@ The `-p` option can be used to skip loading any plugin modules.
 The `-s` option can be used to only parse PELs if they are for serviceable
 errors, while the `-n` option will only parse PELs for non-serviceable ones.
 
+### Installation
+
+If `setuptools` is missing, use this following command to install it.
+```bash
+$ sudo apt-get install python3-setuptools
+```
+The following command can be used for a regular installation.
+```bash
+$ python3 setup.py install
+```
+
+If want to install in [developer mode](https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-e), means changes to the source code will be immediately reflected without needing to reinstall.
```

Comment:
Maybe just add the developer mode description to down below where you mention it already so it's all in one place.

Reply:
Made the edits as requested.